README.md

```diff
@@ -9,17 +18,19 @@ VegBank provides a common storage system and web portal for accessing:
 - Plant taxonomy data
 - Community data
 
+VegBank is an open source, community projects.  We [welcome contributions](./CONTRIBUTING.md) in many forms, including code, graphics, documentation, bug reports, testing, etc.  Use the [DataONE discussions](https://github.com/DataONEorg/dataone/discussions) to discuss these contributions with us.
```

Comment:
Do we want to keep the DataONE discussions link or is there something more appropriate?

Reply:
That's a good question. I will leave this as is, but I opened a ticket (#34)  to update this and replace it with a vegbank-appropriate solution. The README is a template and needs a lot of work still.